charts/pega/templates/_helpers.tpl

```diff
@@ -480,4 +480,20 @@ servicePort: use-annotation
 {{- else -}}
 {{- add .failureThreshold 1 -}}
 {{- end -}}
+{{- end -}}
+
+{{- define "serviceName" -}}
```

Comment:
The name of this template is very generic. Can this be renamed to more contextual?

Reply:
@kishorv10 , updated with hazelcast name